src/core/entities/SkyBox.js

```diff
@@ -60,7 +61,7 @@ export class SkyBox extends MREntity {
             textureNames.map((name) => (path ? path + name : name)),
             this.onTextureLoaded.bind(this) // Ensuring the correct context
         );
-        const geometry = new THREE.SphereGeometry(1000, 32, 16);
+        const geometry = new THREE.SphereGeometry(900, 32, 16);
```

Comment:
just curious on the change to 900, was it a render issue?

Reply:
I shrank down the fulstrum distance from 10000 to 1000, which put this right up on the edge, causing clipping when you walk around.